Jenkinsfile

```diff
@@ -101,31 +101,19 @@ pipeline {
             params.EE_BRANCH_NAME = 'cambpm-ee-main/pipeline-master'
           } else {
             labels = JsonOutput.toJson(pullRequest.labels)
+            shaCE = sh(script:"git rev-parse HEAD", returnStdout: true).trim()
           }
 
           if (cambpmWithLabels('webapp-integration','all-as','h2','websphere','weblogic','jbosseap','run','spring-boot','authorizations')) {
-            build job: "cambpm-ee/${params.EE_BRANCH_NAME}", parameters: [
-                    string(name: 'copyArtifactSelector', value: '<TriggeredBuildSelector plugin="copyartifact@1.45.1">  <upstreamFilterStrategy>UseGlobalSetting</upstreamFilterStrategy>  <allowUpstreamDependencies>false</allowUpstreamDependencies></TriggeredBuildSelector>'),
-                    booleanParam(name: 'STANDALONE', value: false),
-                    string(name: 'CE_BRANCH_NAME', value: "${env.BRANCH_NAME}"),
-                    string(name: 'PR_LABELS', value: labels)
-            ], quietPeriod: 10, wait: false
+            cambpmTriggerDownstream("cambpm-ee/${params.EE_BRANCH_NAME}", labels, shaCE, true, true)
```

Comment:
The method signatures for the downstream branch triggers should be updated if changes to the camunda/cambpm-jenkins-shared-library#7 PR are made.